Sources/Sentry/SentryProfiler.mm

```diff
@@ -490,39 +504,10 @@ - (void)start
     [self startMetricProfiler];
 }
 
-- (void)stop
-{
-    if (_profiler == nullptr) {
-        SENTRY_LOG_WARN(@"No profiler instance found.");
-        return;
-    }
-    if (!_profiler->isSampling()) {
-        SENTRY_LOG_WARN(@"Profiler is not currently sampling.");
-        return;
-    }
-
-    _profiler->stopSampling();
-    [_metricProfiler stop];
-    SENTRY_LOG_DEBUG(@"Stopped profiler %@.", self);
-}
```

Comment:
The preexisting instance method `-[stop]` was combined with the eliminated class method `+[stopProfilerForReason:]` to create the only instance method now needed to stop a profiler: `-[stopForReason:]` on new lines 432-442.